.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```

Comment:
we use yarn, so I don't think this is necessary

Reply:
yarn respects this file too, but you are right it is not necessary assuming users have not overridden their default registry locally.  I had overridden mine, but have recently fixed that.  Will remove